### DISCUSSION TITLE
Spot instance

### DISCUSSION THREAD

**Discussion by KcVamsi** [learndevopsonline/learndevopsonline#456] [Q&A]
Hello Raghu,

When I cancel the spot instance then the on-demand instance is automatically getting terminated and when I try to reuse or restart the spot, I am not getting the option to do. 
I am choosing the persistent type and Interruption Behavior as Stop only.
Please help me if am wrong.

Thank you.

**Comment by noolivara** [comment]
@KcVamsi 

If you cancel the spot request, on demand instance get automatically get terminated.

**"If you only want to stop an On-Demand instance, you do not need to cancel the associated Spot Instance request. Simply stop the On-Demand instance and the Spot Instance request will remain active."**
